src/UserInterfaces/WindowsForms/Controls/MixedCodeDataModel.cs

```diff
@@ -164,9 +167,9 @@ private static Address Align(Address addr, uint alignment)
         /// Preemptively collect the machine code instructions
         /// in all image map blocks.
         /// </summary>
-        private void CollectInstructions()
+        private Dictionary<ImageMapBlock, MachineInstruction[]> CollectInstructions()
         {
-            this.instructions = new Dictionary<ImageMapBlock, MachineInstruction[]>();
+            Dictionary<ImageMapBlock, MachineInstruction[]> instructions = new Dictionary<ImageMapBlock, MachineInstruction[]>();
             foreach (var bi in imageMap.Items.Values.OfType<ImageMapBlock>()
```

Comment:
Style: Prefer using `var instructions....` when creating an object with `new`, especially if the type of the created object is long and awkward.

Reply:
Good catch, I also prefer var ..., Copy-paste mistake I guess :P